layers/oci8/Dockerfile

```diff
@@ -10,27 +10,28 @@ ENV ORACLE_BUILD_DIR=${BUILD_DIR}/oracle
 # Install libaio
 RUN yum install -y libaio
 
+# Instant Client newer than 21.x requires glibc 2.28+ which is not available on Amazon Linux 2
 RUN mkdir -p ${ORACLE_BUILD_DIR}; \
     cd ${ORACLE_BUILD_DIR}; \
-    curl -o oci-basic.zip https://download.oracle.com/otn_software/linux/instantclient/2110000/instantclient-basiclite-linux.x64-21.10.0.0.0dbru.zip && \
-    unzip oci-basic.zip -d src && \
-    curl -o oci-sdk.zip https://download.oracle.com/otn_software/linux/instantclient/2110000/instantclient-sdk-linux.x64-21.10.0.0.0dbru.zip && \
-    unzip oci-sdk.zip -d src
+    curl -o oci-basic.zip https://download.oracle.com/otn_software/linux/instantclient/2116000/instantclient-basiclite-linux.x64-21.16.0.0.0dbru.zip && \
+    unzip oci-basic.zip -d src -x META-INF/* && \
```

Comment:
Could you explain this change?

Reply:
The `-x META-INF/*` excludes this directory from unzip extraction. Seems like it was not there previously and added at some time. These are some metadata like checksums of files, not needed for compilation. The issue is that both oci-basic.zip and oci-sdk.zip are extracted to single output directory `src`. Because there are equally named files in this directory, the unzip of the latter file causes interactive propmt: `replace src/META-INF/MANIFEST.MF? [y]es, [n]o, [A]ll, [N]one, [r]ename:  NULL
(EOF or read error, treating as "[N]one" ...)` and the unzip fails with error.